db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
```

Comment:
When I run the migrations I gewt the following warnings:

Loading application environment...
Loading code in search of Active Record models...
Generating Entity-Relationship Diagram for 39 models...
Warning: Ignoring invalid model ActiveStorage::Blob (table active_storage_blobs does not exist)
Warning: Ignoring invalid model ActiveStorage::VariantRecord (table active_storage_variant_records does not exist)
Warning: Ignoring invalid model ActiveStorage::Attachment (table active_storage_attachments does not exist)
Warning: Ignoring invalid association :taggable on TagTaggable (polymorphic interface Taggable does not exist)
Warning: Ignoring invalid association :container_materials on Ont::Library (Could not find the association :requests in model Ont::Library)
Warning: Ignoring invalid association :source_wells on Ont::Library (undefined method `klass' for nil:NilClass)
Done! Saved diagram to erd.jpg.

Reply:
Warning: Ignoring invalid model ActiveStorage::Blob (table active_storage_blobs does not exist)
Warning: Ignoring invalid model ActiveStorage::VariantRecord (table active_storage_variant_records does not exist)
Warning: Ignoring invalid model ActiveStorage::Attachment (table active_storage_attachments does not exist)

Predate these changes, the others relate to the remains of the Ont libraries, can have a look to see if I can remove the bits that re causing it to get confused though.